### DISCUSSION TITLE
Suggested improvements to Azure Active Directory Authentication docs

### DISCUSSION THREAD

**Discussion by jtv8** [requarks/wiki#4083] [Help / Questions]
The docs at https://docs.requarks.io/auth/azure are now slightly out of date now that Cookie Encryption String has been added as an option. Skipping this setting causes users to be unable to sign in via Google Chrome, and the user-facing error message that results if this happens - "Invalid email / username or password." - is uninformative and difficult to troubleshoot.

After:

> 3. Enter the Identity Metadata Endpoint and Client ID values copied earlier.

I suggest:

> 4. Generate a random 44-character string (any string will do) and copy it into "Cookie Encryption String". (Warning: skipping this step could result in users being unable to sign in via Google Chrome.)

Side note: as a future UX improvement, pre-filling this box with a random string or including a "generate" button would be welcome!

It is also easy to miss the second (important!) part of this step:

> 8. Click on Authentication in the left navigation and enter the Logout URL (https://YOUR-WIKI.DOMAIN.COM) and make sure the ID tokens checkbox under Implicit grant is checked, then click Save at the top.

I suggest splitting this step into two:

> 8. Click on Authentication in the left navigation and enter the Logout URL (https://YOUR-WIKI.DOMAIN.COM).
> 9. Make sure the ID tokens checkbox under Implicit grant is checked, then click Save at the top.

**Comment by ghost** [comment]
After searching for hours how to fix that Google Chrome Cookie problem I finally found this! Thanks for point 4.
